src/apps/mean-vesting/definition.json

```diff
@@ -2762,6 +2968,16 @@
       "code": 6047,
       "name": "CannotModifyTemplate",
       "msg": "Template cannot be modified after streams have been created"
+    },
+    {
+      "code": 6048,
+      "name": "NumberOfIntervalsMustBePossitive",
```

Comment:
@eydelrivero , btw there is a type here NumberOfIntervalsMustBePossitive (Po**s**itive)

Reply:
Good catch! Will fix in a separate issue.